classes/utils/GUI.class.php

```diff
@@ -66,13 +66,15 @@ public static function stylesheets()
             'lib/jquery-ui/jquery-ui.min.css',
             'lib/font-awesome/css/font-awesome.min.css',
             'lib/bootstrap/dist/css/bootstrap.min.css',
-            'lib/flag-icons/css/flag-icons.min.css',            
+            'lib/flag-icons/css/flag-icons.min.css',
             'css/default.css',
+            'css/new-ui/styles.css', // Adding the new-ui styles
+            'css/themes/rnp.css', // Adding the new-ui theme styles
```

Comment:
Is statically adding a theme intentional?

Reply:
I ended up committing the RNP theme by accident. I will remove it.